src/lib/api/worlds.ts

```diff
@@ -56,6 +98,68 @@ export class WorldsAPI extends BaseAPI {
       return null
     }
   }
-}
 
-export const content = new WorldsAPI(WORLDS_CONTENT_SERVER)
+  public getPermissions = async (worldName: string) => {
+    if (!this.authorization) {
+      return null
+    }
+
+    const path = `/world/${worldName}/permissions`
+    const headers = this.authorization.createAuthHeaders('get', path)
+    const result = await fetch(this.url + path, { headers })
+
+    if (result.ok) {
+      const json: WorldPermissionsResponse = await result.json()
+      return json.permissions
+    } else {
+      return null
+    }
+  }
+
+  public postPermissionType = async (
+    worldName: string,
+    worldPermissionNames: WorldPermissionNames,
+    worldPermissionType: WorldPermissionType
+  ) => {
+    if (!this.authorization) {
+      return null
```

Comment:
If there's no authorization shouldn't it throw?

Reply:
WDYT about `throw new Error('Unauthorized')`